apps/frontend/src/components/snapshots/snapshot-plant-information.tsx

```diff
@@ -0,0 +1,68 @@
+
+import PlantObservation from "@/components/observations/plant-observation";
+import { Accordion, AccordionContent, AccordionItem, AccordionTrigger } from "@/components/ui/accordion";
+import { PlantInformation } from "../../../types/observations";
+import PlantObservationForm from "../observations/plant-observation-form";
+
+export default function SnapshotPlantInformation( 
+    {treeData, shrubData, grassData, otherData, editingButtons}: {treeData?: PlantInformation[], shrubData?: PlantInformation[], grassData?: PlantInformation[], otherData?: PlantInformation[], editingButtons: boolean},
+) {
+  return(
+    <div>
+    <h1>
+        Plants
+    </h1>
+    <div className="border border-gray-300 rounded-lg p-4 max-h-[200px] overflow-y-auto">
+      {editingButtons && 
+      <PlantObservationForm newPlant={true} />
+      }
+        <Accordion type="single" collapsible className="w-full">
+            <AccordionItem value="item-1">
+                <AccordionTrigger>Trees</AccordionTrigger>
+                <AccordionContent>
+                
+                  {treeData &&treeData.map((tree, index) => (
```

Comment:
Make sure you have your formatter to format on save to have more consistent syntax